sql/pom.xml

```diff
@@ -180,6 +180,11 @@
       <artifactId>validation-api</artifactId>
       <scope>provided</scope>
     </dependency>
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-api</artifactId>
+      <version>1.7.25</version>
```

Comment:
Can this come from the parent pom ?

Reply:
Hi, I just remembered why I didn't use the version from the parent pom. Their is a parent class variable `<slf4j.version>1.7.12</slf4j.version>`, however the sl4j-api is using a different version in the parent pom, which is also hardcoded.